addons/preview-project-description/userscript.js

```diff
@@ -0,0 +1,68 @@
+export default async function ({ addon, msg }) {
+  const loggedInUser = await addon.auth.fetchUsername();
+  const projectOwner = (await addon.tab.waitForElement(".project-header .avatar")).alt;
```

Comment:
I think it's better to wait until the notes exist above this line, and use this instead

```suggestion
  const projectOwner = addon.tab.redux.state.preview.projectInfo.author.username;
```